src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,95 @@
+const { expensesService, usersService } = require('./../services');
+const { expenseSchema } = require('./../schemas');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const expenses = expensesService.getAll({
+    userId: +userId,
+    categories,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expensesService.getById(+id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.json(expense);
+};
+
+const create = (req, res) => {
+  const { error, value } = expenseSchema.validate(req.body, {
+    stripUnknown: true,
+  });
+
+  const isUserExist = !!usersService.getById(value.userId);
```

Comment:
In the `create` function, ensure that `value.userId` exists before calling `usersService.getById(value.userId)`. If `value.userId` is undefined, it could lead to unexpected behavior.